lib/physiqual/bucket_generators/equidistant_bucket_generator.rb

```diff
@@ -2,25 +2,31 @@ module Physiqual
   module BucketGenerators
     class EquidistantBucketGenerator
       include BucketGenerator
-      def initialize(measurements_per_day, interval, last_measurement_time)
+      def initialize(measurements_per_day, interval, hours_before_first_measurement)
         @measurements_per_day = measurements_per_day
         @interval = interval
-        @last_measurement_time = last_measurement_time
```

Comment:
yeah ik moest het een paar keer lezen, maar dit is toch inderdaad wel nice, en logischer.